core/services/relay/evm/functions/logpoller_wrapper.go

```diff
@@ -239,10 +269,36 @@ func (l *logPollerWrapper) LatestEvents() ([]evmRelayTypes.OracleRequest, []evmR
 		}
 	}
 
-	l.lggr.Debugw("LatestEvents: done", "nRequestLogs", len(resultsReq), "nResponseLogs", len(resultsResp), "nextBlock", nextBlock, "latest", latest)
+	l.lggr.Debugw("LatestEvents: done", "nRequestLogs", len(resultsReq), "nResponseLogs", len(resultsResp), "startBlock", startBlock, "endBlock", latest)
```

Comment:
Let's include start-end blocks for both reqs/resps, how many were originally fetched and how many are left after filtering.